content/articles/wordpress-penetration-testing/index.md

```diff
@@ -126,9 +127,15 @@ To do a simple scan, you can use a graphical interface. To do so, open the Kali
 
 ![Xsser](/engineering-education/wordpress-penetration-testing/xxser.jpg)
 
-When the graphical interface opens, set the suitable options and enter the site's details, and WP pen-testing for XSS vulnerabilities will begin.
+When the graphical interface opens, set the suitable options and enter the site's details, and WP pen-testing for XSS vulnerabilities will begin. To learn more, go through the [Xsser documentation](https://xsser.03c8.net/).
 
 ### Conclusion
 I hope this guide has given an excellent introduction to WordPress penetration testing and practice as well. WordPress security is essential, and it's valuable to invest in the security of your WordPress website.
 
-Using the information provided in this document, you can come up with a WP pen-testing strategy. Now assemble your tools to perform reconnaissance and scanning, exploit where needed and mitigate vulnerabilities that you discover. Good luck!
\ No newline at end of file
+Using the information provided in this document, you can come up with a WP pen-testing strategy. Now assemble your tools to perform reconnaissance and scanning, exploit where needed and mitigate vulnerabilities that you discover. Good luck!
+
+### Further reading
+To learn more about WordPress penetration testing, go through these articles:
++ https://securityboulevard.com/2020/03/penetration-testing-for-wordpress-websites/
++ https://gupta-bless.medium.com/penetration-testing-on-wordpress-f96bc701832e
++ https://www.getastra.com/blog/security-audit/wordpress-penetration-testing/
```

Comment:
```suggestion
To learn more about WordPress penetration testing, go through these articles:
+ https://securityboulevard.com/2020/03/penetration-testing-for-wordpress-websites/
+ https://gupta-bless.medium.com/penetration-testing-on-wordpress-f96bc701832e
+ https://www.getastra.com/blog/security-audit/wordpress-penetration-testing/
```
To learn more about WordPress penetration testing, go through these articles:
+  [Penetration Testing for WordPress]( https://securityboulevard.com/2020/03/penetration-testing-for-wordpress-websites/)
+ [Penetration Testing on WordPress]( https://gupta-bless.medium.com/penetration-testing-on-wordpress-f96bc701832e)
+ [WordPress Penetration Testing](https://www.getastra.com/blog/security-audit/wordpress-penetration-testing/)